src/ann_solo/prosit.py

```diff
@@ -0,0 +1,84 @@
+from typing import Dict, List
+
+import numpy as np
+import tqdm
+import tritonclient.grpc as grpcclient
+
+from ann_solo.config import config
+
+def get_predictions(peptides: List(str), precursor_charges: List(int),

```

Comment:
**question:** The type hints look a bit weird. Shouldn't you use square brackets instead of parentheses?

Reply:
FIxed!